keras/src/utils/saved_model_utils.py

```diff
@@ -0,0 +1,73 @@
+import json
```

Comment:
Can you move the file to `saving/file_editor.py`?

Reply:
I changed the file location.